### PR TITLE
TLS and Authentication

### PR DESCRIPTION
**Pull request by sfudeus**
This PR from @poggenpower adds TLS and auth support to the MQTT client

### CODE REVIEW

File: src/rctmon/mqtt.py [hunk 1]
```diff
@@ -10,20 +10,23 @@
...
-
-    def _connect(self):
-        self.mqtt_client = mqtt.Client()
-        self.mqtt_client.connect(self.mqtt_host, self.mqtt_port)
+        self.conf = mqtt_config
+        self.mqtt_client = self._connect()
+
+    def _connect(self) -> 'MqttClient':
+        mqtt_client = mqtt.Client()
+        if self.conf.user and self.conf.auth_pass:
```
Comment (sfudeus) [inline src/rctmon/mqtt.py]:
```suggestion
        if self.conf.auth_user and self.conf.auth_pass:
```

This would be auth_user according to the config.

Comment (poggenpower) [reply src/rctmon/mqtt.py]: is fixed now. Should have done better testing. Which I hopfuly did now.

Comment (sfudeus) [reply src/rctmon/mqtt.py]: 👍

**Comment by sfudeus** [comment]
lgtm, thx

**Pull request merged**